buffer/src/test/java/io/netty/buffer/AbstractCompositeByteBufTest.java

```diff
@@ -49,6 +49,7 @@
 /**
  * An abstract test class for composite channel buffers
  */
+@SuppressWarnings("deprecation")
```

Comment:
can we do this at a more fine granularity on the methods which cause the warnings?